manim/mobject/svg/tex_mobject.py

```diff
@@ -147,7 +147,9 @@ def __init__(self, *tex_strings, **kwargs):
         SingleStringTexMobject.__init__(
             self, self.arg_separator.join(tex_strings), **kwargs
         )
-        self.break_up_by_substrings()
+        config = dict(self.CONFIG)
```

Comment:
Comment: this is overriding the global config dict. Not sure if it's a problem, just wanted to raise awareness.

Reply:
I think this was deliberate - but has been moved to a better spot.
This had to do with the bug that cause all LaTeX strings to be compiled twice. Once with alignment "" and once with alignment "\\centering".